apps/updatenotification/lib/Controller/AdminController.php

```diff
@@ -107,4 +122,35 @@ public function createCredentials(): DataResponse {
 
 		return new DataResponse($newToken);
 	}
+
+	// Copied from https://github.com/nextcloud/server/blob/a06001e0851abc6073af678b742da3e1aa96eec9/lib/private/Support/Subscription/Registry.php#L187-L214
+	private function getUserCount(): int {
+		$userCount = 0;
+		$backends = $this->userManager->getBackends();
+		foreach ($backends as $backend) {
+			if ($backend->implementsActions(Backend::COUNT_USERS)) {
```

Comment:
```suggestion
			if ($backend instanceof ICountUsersBackend) {
```

Then you can remove the PHPDoc annotation.

Reply:
Not really because not all the backend explicitly extends ICountUsersBackend. This should be changed and I already adapted a few days ago user_ldap, but there is probably more to do

Reply:
We can at least add this as a `TODO` comment

Reply:
Will add a todo 👍

Reply:
done